### ISSUE TITLE
Sandcastle UX improvements

### ISSUE THREAD

**Issue by rahwang** [AnalyticalGraphicsInc/cesium#5270]
Several of the most useful Sandcastle features are non-obvious. For instance, many users don't notice the search tool, the share button or the documentation links (I didn't know about that one until today!). 

Worth thinking about how to make these more prominent. I'll take point on this one.

**Comment by emackey** [comment]
Yep.  Would be great to move the whole UI framework off of Dojo, too.

Also, most other live editors present the HTML, JS, and CSS windows simultaneously with the result window, so you can see all the parts at once.  It would be nice if we could do something like that.

**Comment by pjcozzi** [comment]
+1 for moving off of Dojo but it is not in our near-term scope.

I also don't think the HTML/CSS is important enough for most of our cases to justify the screen real estate.

**Comment by ggetz** [comment]
Closing as OBE.
